### PR TITLE
refactor: emit cumulative fees when incentivizing a packet multiple times

### PR DESCRIPTION
**Pull request by colin-axner**
## Description

closes: #1327 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [x] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [x] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [x] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [x] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Review by crodriguezvega — APPROVED** [review]

**Review by AdityaSripal — APPROVED** [review]
nice and clean!

File: modules/apps/29-fee/keeper/events.go [hunk 1]
```diff
@@ -9,17 +9,33 @@ import (
...
+		totalAckFees     sdk.Coins
+		totalTimeoutFees sdk.Coins
+	)
+
+	for _, fee := range packetFees.PacketFees {
+		// only emit total fees for packet fees which allow any relayer to relay
+		if fee.Relayers == nil {
+			totalRecvFees.Add(fee.Fee.RecvFee...)
+			totalAckFees.Add(fee.Fee.AckFee...)
+			totalTimeoutFees.Add(fee.Fee.TimeoutFee...)
```
Comment (damiannolan) [inline modules/apps/29-fee/keeper/events.go:26]: Should we reassign as [`coins.Add(...)` returns `sdk.Coins`](https://github.com/cosmos/cosmos-sdk/blob/main/types/coin.go#L303)? 

```
totalRecvFees = totalRecvFees.Add(...)
```

I think this needs to be done for the singular [`coin.Add()`](https://github.com/cosmos/cosmos-sdk/blob/main/types/coin.go#L105) function. Not 100% sure about `Coins` but may be good practise to keep consistent and handle return value.

Comment (colin-axner) [reply modules/apps/29-fee/keeper/events.go:26]: seems like a good idea to me

Comment (colin-axner) [reply modules/apps/29-fee/keeper/events.go:26]: nice catch :bug: - the tests were using an expected value of "", the lack of reassigning didn't change the variable values

**Review by seantking — APPROVED** [review]

**Pull request merged**